include/esb.h

```diff
@@ -127,6 +127,11 @@ enum esb_bitrate {
 	/** 4 Mb radio mode. */
 	ESB_BITRATE_4MBPS = NRF_RADIO_MODE_NRF_4MBIT_H_0_5,
 #endif /* defined(RADIO_MODE_MODE_Nrf_4Mbit0_5) */
+
+#if defined(RADIO_MODE_MODE_Nrf_4Mbit_0BT6)
+	/** 4 Mb radio mode. */
+	ESB_BITRATE_4MBPS = RADIO_MODE_MODE_Nrf_4Mbit_0BT6,
+#endif /* defined(RADIO_MODE_MODE_Nrf_4Mbit_0BT6) */
```

Comment:
Those two `#if defined` conditions defines single enum item with a different value. Are the conditions always mutually exclusive?

Reply:
`RADIO_MODE_MODE_Nrf_4Mbit0_5` exists for the nRF54H20, while `RADIO_MODE_MODE_Nrf_4Mbit_0BT6` exists for the nRF54L15.